src/org/omegat/MainClassLoader.java

```diff
@@ -73,13 +62,12 @@ public MainClassLoader() {
      * Main class can add jar classpath for plugins in dynamic manner.
      * @param url
      */
-    synchronized void add(URL url) {
+    public synchronized void add(URL url) {
```

Comment:
Should not be `synchronized` to avoid dead-lock. 
```suggestion
    public  void add(URL url) {
```

Reply:
It would be better to change method name to `addJarToClasspath(URL)` as same as  a `addJartoClasspath(String)` and `addJarToClasspath(File)`